README.md

```diff
@@ -47,7 +47,7 @@ Or you can run it as a normal Java application:
 Description | Default
 ---------------------------------------------------------------|-------------------
 server.port                                                    | 8080
-pitchfork.forwarders.haystack.kafka.enabled                    | true
```

Comment:
Reason for changing this: There's multiple options now (Kafka + Kinesis) so maybe it's better to turn everything off by default instead of trying to guess what the most common use case will be.

Reply:
fail fast if all forwarders are off?

Reply:
It does fail fast but not on purpose (the injection of the forwarders fail and Spring throws an error).
I'll add a check somewhere so that it fails with an error message that is clear on what the problem is.